DiscordBot.Database/DatabaseService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using DiscordBot.Domain.Configuration;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Options;
+
+namespace DiscordBot.Domain.Database

```

Comment:
https://github.com/coderdojo-linz/DiscordBot-DotNet/pull/56#discussion_r452138728

Reply:
But what's this: https://github.com/coderdojo-linz/DiscordBot-DotNet/blob/master/DiscordBot.CoderDojoInfoModule/CoderDojoInfoModule.cs#L11 ? There is also code.

Reply:
I dont know. Must be a mistake